src/main/java/edu/ie3/datamodel/io/naming/timeseries/IndividualTimeSeriesMetaInformation.java

```diff
@@ -3,14 +3,14 @@
  * Institute of Energy Systems, Energy Efficiency and Energy Economics,
  * Research group Distribution grid planning and operation
 */
-package edu.ie3.datamodel.io.csv.timeseries;
+package edu.ie3.datamodel.io.naming.timeseries;
 
-import edu.ie3.datamodel.io.csv.FileNameMetaInformation;
+import edu.ie3.datamodel.io.naming.DataSourceMetaInformation;
 import java.util.Objects;
 import java.util.UUID;
 
 /** Specific meta information, that can be derived from a individual time series file */
-public class IndividualTimeSeriesMetaInformation extends FileNameMetaInformation {
+public class IndividualTimeSeriesMetaInformation extends DataSourceMetaInformation {
```

Comment:
I know, it's not your fault (I introduced this flaw), but I'm not super happy with the fact, that this is not an abstract class. It would be better, if you could only instantiate `SqlIndividualTimeSeriesMetaInformation` and `CsvIndividualTimeSeriesInformation`.

One opportunity, I would like you to check out, is to check, if `EntityPersistenceNamingStrategy#extractIndividualTimesSeriesMetaInformation` couldn't be moved to both of the implementing classes as static method `parse(String, ...)`. Parts of the logic of the method could obviously be moved here and then the individual time series pattern doesn't have to be publicly accessible.

What do you think?

Reply:
To the second part: A counter-argument would be here, that `EntityPersistenceNamingStrategy` can exist and thus allow for different naming strategies of files/tables. When integrating the function into the timeseries meta information, only one single naming strategy could exist.

Reply:
To the first point: `IndividualTimeSeriesMetaInformation` is currently instantiated in `EntityPersistenceNamingStrategy#extractIndividualTimesSeriesMetaInformation` and I think it makes some sense that way. If we want to make the class abstract, we have to think of another solution there.